backend/src/api/routes/auth/index.routes.ts

```
@@ -721,6 +735,96 @@ router.get(
+      newPassword,
+      oldPassword,
+      req.user.id
+    );
+    successResponse(res, result);
+  } catch (error) {
+    next(error);
+  }
+})
+
+//GET all admins to show
+router.get("/admin/allAdmins", verifyAdmin, async (req: Request, res: Response, next: NextFunction) => {
```

ESLint

[error] Replace `"/admin/allAdmins",·verifyAdmin,` with `⏎··'/admin/allAdmins',⏎··verifyAdmin,⏎·`

(prettier/prettier)